go-package.sh

```
@@ -0,0 +1,61 @@
+# The script must be started in the root directory of the project
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)